tests/describe.spec.js

```diff
@@ -9,25 +9,24 @@ describe('describe', () => {
             meta: 'cyborg'
         });
 
-        return carotte.invoke('hello-to-you-describe!:describe', {})
+        return carotte.invoke('hello-to-you-describe!:describe:v2', {})
             .then(data => {
                 expect(data.meta).to.be.a('string');
                 expect(data.meta).to.eql('cyborg');
             });
     });
 
-    it('should expose metas on a topic route using a :describe suffix on a direct route', (done) => {
+    it('should expose metas on a topic route using a :describe suffix on a direct route', () => {
```

Comment:
```suggestion
    it('should expose metas on a topic route using a :describe:v2 suffix on a direct route', () => {
```

there is another one just above, but GitHub won't let me comment it 😬